eZ/Publish/Core/FieldType/Tests/RichText/Converter/Xslt/_fixtures/xhtml5/output/018-htmlTable.xml

```diff
@@ -4,13 +4,13 @@
     <caption>Table caption.</caption>
     <tbody>
       <tr class="rowClass1">
-        <th class="headingClass1" colspan="1" rowspan="5" abbr="XSLT" scope="col" style="width:102px;vertical-align:top;">11</th>
+        <th class="headingClass1" colspan="1" rowspan="5" abbr="XSLT" scope="col" style="width:102px;vertical-align:top;">11<br/>with some line break</th>
         <th class="headingClass2" colspan="2" rowspan="6" abbr="XSD" scope="row" style="width:37%;vertical-align:middle;">12</th>
       </tr>
       <tr class="rowClass2">
-        <td class="cellClass1" colspan="3" rowspan="7" style="width:74%;vertical-align:bottom;">21</td>
+        <td class="cellClass1" colspan="3" rowspan="7" style="width:74%;vertical-align:bottom;">21<br/>with some line break</td>
         <td class="cellClass2" colspan="4" rowspan="8" style="width:38px;vertical-align:baseline;">22</td>
       </tr>
     </tbody>
   </table>
-</section>
+</section>
```

Comment:
missing Line Feed.

Reply:
Nitpick fixed. I'll open PR in [`ezsystems/ezplatform-richtext`] as you suggested.